test_common/harness/kernelHelpers.cpp

```diff
@@ -614,6 +614,26 @@ static int get_offline_compiler_output(
             }
         }
     }
+
+    if (compilationMode == kSpir_v && !gDisableSPIRVValidation)
+    {
+        std::string runString = gSPIRVValidator + " " + outputFilename;
```

Comment:
Should we be specifying the SPIR-V version/environment to validate against here (i.e. `--target-env=...`)? By default SPIRV-Tools will validate against SPIR-V 1.5, which may let some things through that are not valid for the version of SPIR-V actually being tested.

Reply:
I'd be fine filing an issue to improve this and saving this change for a subsequent PR.

I guess if we wanted to pick the target SPIR-V environment the "right way" to do that would be to query for the OpenCL version supported by the device?  Note, we still need to add the OpenCL 3.0 environment to the validator...

Reply:
I'd be happy to raise an issue, although I don't have a very clear picture of what the "right way" of handling this is, here, so I'd be equally happy to defer to someone with a clearer picture of this.